core/src/main/java/org/mskcc/cbio/oncokb/apiModels/TumorType.java

```diff
@@ -13,15 +15,25 @@
 @ApiModel(description = "OncoTree Detailed Cancer Type")
 public class TumorType implements Serializable {
 
+    @ApiModelProperty(value = "Database TumorType ID")
     private Integer id = null;
+    @ApiModelProperty(value = "Oncotree code")
     private String code = null;
+    @ApiModelProperty(value = "(Nullable) Color of tumor")
     private String color = null;
+    @ApiModelProperty(value = "(Nullable) Oncotree sub type")
     private String name = null;
+    @ApiModelProperty(value = "(Nullable) Oncotree tumor main type")
     private MainType mainType = null;
+    @ApiModelProperty(value = "(Nullable) Oncotree tumor tissue")
     private String tissue = null;
+    @ApiModelProperty(value = "Map of child tumor types")
     private Map<String, TumorType> children = new HashMap<String, TumorType>();
+    @ApiModelProperty(value = "(Nullable) Parent tumor name")
     private String parent = null;
+    @ApiModelProperty(value = "Oncotree tumor type level. -1 indicates special tumor types")
```

Comment:
Will they know what a special tumor type is?

Reply:
Good point, I can point them to SpecialTumorTypes.java file